ci-operator/step-registry/interop-tests/deploy-odf/OWNERS

```diff
@@ -1,17 +0,0 @@
-# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```

Comment:
We can't delete the interop-tests directory in this PR as this is not our directory. If you remove any changes to these files then they will just not appear in this PR anymore.

Reply:
This was removed by mistake and the directory has been restored.